runtime/compiler/il/symbol/J9Symbol.cpp

```diff
@@ -271,6 +271,44 @@ J9::Symbol::getRecognizedField()
       return TR::Symbol::UnknownField;
    }
 
+/**
+ * Return the owning class name of this recognized field.
+ * Return null if this symbol does not have a recognized field.
+ */
+const char *
+J9::Symbol::owningClassNameCharsForRecognizedField(int32_t & length)
```

Comment:
Is there some kind of assert to make sure that we are calling this on a field of some kind? is there any kind of assertion we should have to make sure we find the recognizedField for something that should be recognized?

Reply:
Added assertion to make sure the receiver symbol is 1) a shadow symbol, and 2) has a valid recognized field.